src/Common/Constants.ts

```diff
@@ -131,6 +130,10 @@ export class Features {
   public static readonly enableSDKoperations = "enablesdkoperations";
 }
 
+export class Flights {
+  public static readonly settingsV2 = "settingsv2";

```

Comment:
>settingsv2 [](start = 39, length = 10)

Settingsv2? Can be the flight name with upper case?

Reply:
resolved offline, flights returned from experimentation package are lowercase